readme.md

```diff
@@ -0,0 +1,120 @@
+[![Build Status](https://travis-ci.org/ninech/ordy.svg)](https://travis-ci.org/ninech/ordy)
+
+## Usage
+
+#### Ordering
+
+__Model usage__
+
+```rb
+class User < ApplicationRecord
+  include Ordy::Orm::ActiveRecord::Orderable
+  
+  has_many :comments
+
+  orderable_by do
+    # order by field
+    columns :name, :email
+    
+    # order by relations field
+    associations comments: :scripts
+    
+    # sort by specified order
+    specified(state: %w(new pendinguser migrating failed))
+    
+    # custom query
+    query :users do |scope, args|
```

Comment:
It doesn't make much sense to order `User` by `:users`, don't you agree?
Also give some real example for `where` and `order` method arguments.